tools/plugins/com.liferay.ide.hook.core/src/com/liferay/ide/hook/core/operation/NewHookDataModelProvider.java

```diff
@@ -234,6 +239,26 @@ public void init()
     @Override
     public IStatus validate( String propertyName )
     {
+        boolean is62 = false;
+
+        try
+        {
+            SDK sdk = SDKUtil.getWorkspaceSDK();
+
+            if( sdk != null )
+            {
+                final Version version = new Version( sdk.getVersion() );
+                final Version sdk62 = ILiferayConstants.V620;
+                if( CoreUtil.compareVersions( version, sdk62 ) == 0 )
```

Comment:
This should work fine on 6.1 as well, so we should just do a check to make sure that the version is not greater or equal to 7.0 instead.